app/controllers/BookingController.scala

```diff
@@ -15,29 +15,63 @@ import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.io.Source
 
 class BookingController @Inject()(
-                                 components: ControllerComponents,
-                                 val reactiveMongoApi: ReactiveMongoApi,
-                                 val bookingServices: BookingServices,
-                                 val releasedServices: ReleasedServices
-                               ) extends AbstractController(components)
+                                   components: ControllerComponents,
+                                   val reactiveMongoApi: ReactiveMongoApi,
+                                   val bookingServices: BookingServices,
+                                   val releasedServices: ReleasedServices
+                                 ) extends AbstractController(components)
   with MongoController with ReactiveMongoComponents with play.api.i18n.I18nSupport {
 
   implicit def ec: ExecutionContext = components.executionContext
 
-  def showForm = Action {implicit request: Request[AnyContent] =>
-    Ok(views.html.booking(Booking.createBookingForm))
+  def showForm = Action.async { implicit request: Request[AnyContent] =>
+    titlesAndScreenings.map { titles =>
+      Ok(views.html.bookingfilm(Booking.getTitleForm, titles))
+    }
   }
 
-  def submitForm = Action.async {implicit request: Request[AnyContent] =>
-    Booking.createBookingForm.bindFromRequest.fold({formWithErrors =>
+  def submitSelectFilmForm(title: String) = Action.async { implicit request: Request[AnyContent] =>
+    titlesAndScreenings.map { titles =>
+
+      Ok(views.html.booking(Booking.createBookingForm,
+        title,
+        titles.filter(thing =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of head on collections.](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5277100)